libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/ranker/SimpleWeightedSumDataLayoutStrategyScorer.java

```diff
@@ -0,0 +1,83 @@
+package com.linkedin.openhouse.datalayout.ranker;
+
+import com.linkedin.openhouse.datalayout.strategy.DataLayoutStrategy;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Optimizing across multiple objective such as (x) Maximize file count reduction (x) Minimize
+ * compute cost used for compaction by first normalizing the objectives on a common scale and then
+ * performing scalarization using a weighted sum technique.
+ */
+public class SimpleWeightedSumDataLayoutStrategyScorer implements DataLayoutStrategyScorer {
+
+  public static final int MINIMIZATION_OBJECTIVE_FACTOR = -1;
```

Comment:
What is this used for?

Reply:
Just for code readability, but open to suggestions here. 

In weighted sum, a minimization objective (obj2) is 
`w1 * obj1 - w2 * obj2`

In code, writing it as 
`w1 * obj1 + (-1) * w2 * obj2`